index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <!-- meta tag for responsiveness, imported google font styles -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Hina+Mincho&family=Kaisei+Opti&display=swap" rel="stylesheet">
+    <title>Project 2: The News Site - Ocean News</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="hero">
+      <!--  Logo and navbar -->
+      <nav class="navigation">
+        <img class="logo" src="https://cdn.pixabay.com/photo/2015/09/09/17/06/jellyfish-931886_960_720.jpg" alt="Jellyfish logo">
+        <h1>Ocean news</h1>  
+        <div>
+        <a class="link" href="#big-news-link">Big news</a>
+        <a class="link" href="#other-news-link">Other news</a>
+        <a class="link" href="#">About</a>
+        <a class="link" href="#">Contact</a>
+      </div>      
```

Comment:
Thinking you could try and give the div a class and styled it as position: flex in CSS to get "contact" on the same row as the others. Either you could style just the a, if you don't need the class=link.